new-content/events/summer-school-2021.yaml

```diff
@@ -0,0 +1,259 @@
+metadata:
+  title: 'Qiskit Global Summer School 2021'
+  description: 'The Qiskit Global Summer School 2021 is a two-week intensive summer school designed to empower the next generation of quantum researchers and developers with the skills and know-how to explore quantum applications on their own'
+  image: '/images/events/physics-of-computation/card-image-endicott-house-1981.png'
+
+header:
+  primaryTitle: 'Qiskit Global Summer School 2021:'
```

Comment:
IMO calling these "primary" and "secondary" titles is a little confusing, since it implies some kind of either hierarchy or importance, but in this case it's all about presentation.

Probable "titleLine1" and "titleLine2" of something like that might be more straight forward.